thirdeye/thirdeye-pinot/src/test/java/com/linkedin/thirdeye/detector/metric/transfer/testMetricTransfer.java

```diff
@@ -0,0 +1,68 @@
+package com.linkedin.thirdeye.detector.metric.transfer;
+
+import com.linkedin.thirdeye.api.MetricSchema;
+import com.linkedin.thirdeye.api.MetricTimeSeries;
+import com.linkedin.thirdeye.api.MetricType;
+import com.linkedin.thirdeye.api.TimeRange;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+import org.testng.annotations.Test;
+import org.testng.Assert;
+
+
+public class testMetricTransfer {
+
+  // apply the
+  @Test
+  public void transfer(){
+
+    // create a mock MetricTimeSeries
+    List<String> names = new ArrayList<>(1);
+    String mName = "metric0";
+    names.add(0, mName);
+    List<MetricType> types = Collections.nCopies(names.size(), MetricType.DOUBLE);
+    MetricSchema _metricSchema = new MetricSchema(names, types);
+    MetricTimeSeries _metrics = new MetricTimeSeries(_metricSchema);
+    double [] m0 = {1.0, 1.0, 1.0, 1.0, 1.0, 1.0};
+    for (long i=0l; i<=5l; i++) {
+      _metrics.set(i, mName, 1.0);
```

Comment:
You might want to check if our naming convention allows "_"

@kishoreg might be able to answer this question.